node/db/Cargo.toml

```diff
@@ -15,7 +15,8 @@ lz4    = ["rocksdb/lz4"]
 zlib   = ["rocksdb/zlib"]
 bzip2  = ["rocksdb/bzip2"]
 # not included by default to reduce build time
-zstd = ["rocksdb/zstd"]
+# uncomment when it needs to be used by other crates
+# zstd = ["rocksdb/zstd"]
```

Comment:
Without this feature, how do we enable `zstd` support in rocksdb?

Reply:
This feature is not yet propagated to daemon. It would just slow down cargo doc --all-features atm. I assume some offline measurement of different compression types is going on and we will eventually keep only one